### PR TITLE
Develop first cut of front-end project layout. Edit auth error reporting.

### PR DESCRIPTION
**Pull request by karthikv**
It's time to begin work on the front-end of napkin.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@
 log
 node_modules
 local.json
+public/stylesheets/main.css
```
Comment (seanmonstar) [inline .gitignore:5]: why ignore the stylesheet?

Comment (karthikv) [inline .gitignore:5]: I'm using stylus, so the real stylesheet is main.styl; it gets compiled to main.css, and there's no point in having both tracked by git. In fact, this just causes unnecessary merge conflicts later on.

File: public/stylesheets/bootstrap-stripped.css [hunk 2]
```diff
@@ -0,0 +1,668 @@
+/*!
```
Comment (karthikv) [inline public/stylesheets/bootstrap-stripped.css:1]: As a quick note, I stripped bootstrap to only what I'm currently using or what I will certainly be using in the future; this is not the original.

**Comment by seanmonstar** [comment]
other then that, i dont see anything glaring

**Pull request merged**